src/main/webapp/ts/profileBuild.ts

```diff
@@ -0,0 +1,19 @@
+function setForm(value: any) {
+    const studentForm =  document.getElementById('student');
+    const professionalForm = document.getElementById('professional')
+
+    if(studentForm && professionalForm){
+            if(value == 'student'){
+                studentForm.setAttribute('style','display:block;');
+                // .style='display:block;';
```

Comment:
I personally prefer the .style = 'display:none;' you have commented out here (L8 and L10), but either is fine. Be sure to clean these up before merging

Reply:
Yea, I had to do .setAttribute bc I kept getting a error in TS. So I'll just clean the comments.
Done.